src/models/Organization.ts

```diff
@@ -125,9 +119,9 @@ const organizationSchema = new Schema({
       ref: "User",
     },
   ],
-  visibleInSearch: {
-    type: Boolean,
-  },
+  // visibleInSearch: {
```

Comment:
remove the code instead of just commenting it, same goes all the instances in this PR

Reply:
I have removed the code